vendor/plist-to-html/plist-to-html/PlistToHtml

```diff
@@ -0,0 +1,173 @@
+    clike = open(os.path.join(layout_dir, 'clike.min.js'), 'r')
+    bug_viewer = open(os.path.join(layout_dir, 'bugviewer.js'), 'r')
+
+    html_output.write('<script type="text/javascript">')
+    html_output.write(codemirror_js.read() + '\n')
+    html_output.write(clike.read() + '\n')
+    html_output.write(bug_viewer.read() + '\n')
+    html_output.write('var data = ' + json.dumps(data) + ';')
+    html_output.write('</script>')
+
+    html_output.write(html_footer.read())
+
+    print('Html file was generated: ' + html_output_path)
+
+
+def parse(file_path, output_path, layout_dir):
+    """
+    Prints the results in the given file to the standard output in a human-
+    readable format.
+    """
+
+    if not file_path.endswith(".plist"):
+        print("Skipping input file '" + file_path + "' as it is not a plist.")
+        return
+
+    print("Parsing input file '" + file_path + "'")
+
+    try:
+        plist = plistlib.readPlist(file_path)
+        generate_html_from_plist(file_path, plist, layout_dir, output_path)
+    except ExpatError as err:
+        print('Failed to process plist file: ' + file_path +
+              ' wrong file format?', err)
+    except AttributeError as ex:
+        print('Failed to get important report data from plist.', ex)
+    except IndexError as iex:
+        print('Indexing error during processing plist file ' + file_path, iex)
+    except Exception as ex:
+        print('Error during processing reports from the plist file: ' +
+              file_path, ex)
+
+
+def add_arguments_to_parser(parser):
+    parser.add_argument('-i', '--input',
+                        type=str,
+                        nargs='*',
+                        metavar='file/folder',
+                        help="The plist files and/or folders containing "
+                             "analysis results which should be parsed.")
+
+    parser.add_argument('-o', '--output',
+                        dest="output_dir",
+                        required=True,
+                        help="Generate HTML output files in the given folder.")
+
+    parser.add_argument('-l', '--layout',
+                        dest="layout_dir",
+                        required=False,
+                        default=os.path.join(os.path.abspath(__file__),
+                                             '..',
+                                             'dist'),
+                        help="Directory which contains dependency html, css "
+                             "and javascript files.")
+
+
+def main():
+    """
+    Plist parser main command line.
+    """
+    parser = argparse.ArgumentParser(
+        prog="PlistToHtml",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    add_arguments_to_parser(parser)
+
+    args = parser.parse_args()
+
+    if isinstance(args.input, str):
+        args.input = [args.input]
+
+    output_dir = os.path.abspath(args.output_dir)
+    if not os.path.exists(output_dir):
```

Comment:
Should we warn here if the directory already exists? Should we clean the previous results?